parse_json_amd64.go

```diff
@@ -28,7 +28,7 @@ import (
 func (pj *internalParsedJson) initialize(size int) {
 	// Estimate the tape size to be about 15% of the length of the JSON message
 	avgTapeSize := size * 15 / 100
-	if cap(pj.Tape) < avgTapeSize {
+	if cap(pj.Tape) <= avgTapeSize {
```

Comment:
@klauspost Was this an intentional change? Allocating a new tape with a cap of avgTapeSize when the existing tape has a cap of avgTapeSize doesn't seem desirable.

Reply:
@vcabbage You are right. Let me test it out.